extensions/sql-database-projects/src/tools/buildHelper.ts

```diff
@@ -7,12 +7,17 @@ import * as vscode from 'vscode';
 import * as path from 'path';
 import { promises as fs } from 'fs';
 import * as utils from '../common/utils';
-import { errorFindingBuildFilesLocation } from '../common/constants';
-import * as mssql from 'mssql';
-import * as vscodeMssql from 'vscode-mssql';
 import * as sqldbproj from 'sqldbproj';
+import * as extractZip from 'extract-zip';
+import * as constants from '../common/constants';
+import { HttpClient } from '../common/httpClient';
 
 const buildDirectory = 'BuildDirectory';
+const sdkName = 'Microsoft.Build.Sql';
+const microsoftBuildSqlVersion = '0.1.3-preview'; // TODO: have this be configurable
```

Comment:
Looks good to me. Only concern I have is the DacFx version coming from STS today is much newer than the one we shipped with SDK v0.1.3-preview. Would this be an issue?

Reply:
Actually that probably means we're overdue for a new version of the SDK :)

Reply:
yeah it's a small concern, but should be less risky if we have a new version of the SDK before the October release. 

Building with one version of DacFx in the SDK and a different version of DacFx/sqlpackage used to publish is also something pipelines will have to deal with. Not sure how much of an issue this'll be in the future as more features/fixes get added in DacFx and if people don't update the SDK version.